cypresse2e/cypress/support/commands.js

```diff
@@ -0,0 +1,25 @@
+// ***********************************************
+// This example commands.js shows you how to
```

Comment:
We could probably remove this example file ?

Reply:
Hi @idjevm 
This file will be use to to define or overwrite commands. This file is loaded before any test files are evaluated.